### PR TITLE
Fix crash for missing owner id of track

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

This bug was seen on prod & crashing user session. Adding console.error code instead of throwing in case somehow the track is not provided in the playlist body and no owner_id is found.

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.

Locally vs. prod with a broken account

### How will this change be monitored?

For features that are critical or could fail silently please describe the monitoring/alerting being added.

New sentry error catching

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-fix-crash

File: src/common/store/errors/sagas.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ function* handleError(action: errorActions.HandleErrorAction) {
           console.debug(
             `Additional error info: ${JSON.stringify(action.additionalInfo)}`
           )
-          scope.setExtras(action.additionalInfo)
+          scope.setExtras({ extras: action.additionalInfo })
```
Comment (jowlee) [inline src/common/store/errors/sagas.ts:32]: Not sure if you need to wrap it in another object - I think it's an empty obj by default. 
Doc: https://getsentry.github.io/sentry-javascript/classes/hub.scope.html#setextras

File: src/common/store/cache/collections/selectors.ts [hunk 2]
```diff
@@ -89,9 +89,18 @@ export const getTracksFromCollection = (
   const ids = collection.playlist_contents.track_ids.map(t => t.track)
   const tracks = getTracks(state, { ids })
 
-  const userIds = Object.keys(tracks).map(
-    id => tracks[(id as unknown) as number].owner_id
-  )
+  const userIds = Object.keys(tracks)
+    .map(id => {
+      const track = tracks[(id as unknown) as number]
+      if (track && track.owner_id) {
```
Comment (jowlee) [inline src/common/store/cache/collections/selectors.ts:95]: could do track?.owner_id if you wanted

**Review by jowlee — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-fix-crash

File: src/common/store/errors/sagas.ts (hunk 1 above)
Comment (raymondjacobson) [reply src/common/store/errors/sagas.ts:32]: thx sry i made this change b/c of a type issue, but this was the wrong change. fixed.!! ty

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-fix-crash

**Pull request merged**